src/psyclone/transformations.py

```diff
@@ -1634,7 +1634,8 @@ class OMPParallelTrans(ParallelRegionTrans):
     # The types of node that this transformation can enclose
     valid_node_types = (psyGen.Loop, psyGen.Kern, psyGen.BuiltIn,
                         psyGen.OMPDirective, psyGen.GlobalSum,
-                        psyGen.Literal, psyGen.Reference)
+                        psyGen.Literal, psyGen.Reference,
+                        psyGen.Assignment, psyGen.BinaryOperation)
```

Comment:
Why have Assignment and BinaryOperation been added here but the rest of the file is unchanged?

Reply:
I needed this for the test case to work - atm in psyir we can not handle a GOKern, so on Sergi's advice I've replaced this with an assignment (which psyir can already handle). There is a #TODO in place to track this.

Reply:
Oh I see - I was being dense :-)